ERCS/erc-7656.md

```diff
@@ -17,7 +18,7 @@ This proposal defines a factory capable of deploying generic services linked to
 
 ## Motivation
 
-Existing standards like [ERC-6551](./eip-6551.md) successfully bind smart accounts to NFTs, allowing registries to deploy accounts owned by specific token IDs. However, these standards have two key limitations:
+Existing proposals like TokenBound Account (numbered as 6551) successfully bind smart accounts to NFTs, allowing registries to deploy accounts owned by specific token IDs. However, these standards have two key limitations:
```

Comment:
Changing the way you write the reference doesn't change the fact that you're making a reference. Either remove it entirely or help get ERC-6551 across the finish line.